sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -40,7 +40,7 @@
 def test_error_messages_on_wrong_input():
     for score_func in score_funcs:
         expected = (
-            r"Found input variables with inconsistent numbers " r"of samples: \[2, 3\]"
+            r"Found input variables with inconsistent numbers of samples: \[2, 3\]"
```

Comment:
same here, raw string w/o escape characters is nicer.

Reply:
I did not remove the `r`, just merged two raw strings into a single raw string.